docs/user/alerting/view-alerts.asciidoc

```diff
@@ -0,0 +1,80 @@
+Refer to <<alerting-security,Alerting security requirements>>.
+====
+
+[discrete]
+[[filter-alerts]]
+=== Filter alerts
+
+preview::[]
+
+In *{stack-manage-app}* > *Alerts*, you can filter the list (for example, by alert status or rule type) and customize the filter controls.
+To search for specific alerts, use the KQL bar to create structured queries using {kibana-ref}/kuery-query.html[{kib} Query Language].
+
+By default, the list contains all the alerts that you have authority to view in the selected time period except those associated with Security rules.
+To view alerts for Security rules, click the query menu and select *Security rule types*:
+
+[role="screenshot"]
+image::images/stack-management-alerts-query-menu.png[The Alerts page with the query menu open]
+// NOTE: This is an autogenerated screenshot. Do not edit it directly.
+
+Alternatively, view those alerts in the {security-guide}/alerts-ui-manage.html[{security-app}].
+
+[discrete]
+[[view-alert-details]]
+=== View alert details
+
+To get more information about a specific alert, open its action menu (…) and select *View alert details* in either *{stack-manage-app} > Alerts* or *{rules-ui}*.
+There you'll see the current status of the alert, its duration, and when it was last updated.
+To help you determine what caused the alert, there is information such as the expected and actual threshold values and a summarized reason for the alert.
+
+If an alert is affected by a maintenance window, the alert details include its identifier.
+For more information about their impact on alert notifications, refer to <<maintenance-windows>>.
+
+[discrete]
+[[alert-status]]
+==== Alert statuses
+
+There are four common alert statuses:
+
+`active`:: The conditions for the rule are met and actions should be generated according to the notification settings.
+`flapping`:: The alert is switching repeatedly between active and recovered states.
+`recovered`:: The conditions for the rule are no longer met and recovery actions should be generated.
+`untracked`:: Actions are no longer generated. For example, you can choose to move active alerts to this state when you disable or delete rules.
+
+[NOTE]
+====
+The `flapping` state is possible only if you have enabled alert flapping detection in *{stack-manage-app} > {rules-ui} > Settings*. For each space, you can choose a look back window and threshold that are used to determine whether alerts are flapping. For example, you can specify that the alert must change status at least 6 times in the last 10 runs. If the rule has actions that run when the alert status changes, those actions are suppressed while the alert is flapping.
+====
+
+
+[discrete]
+[[mute-alerts]]
+=== Mute alerts
+
+If an alert is active or flapping, you can mute it to temporarily suppress future actions.
+In both *{stack-manage-app} > Alerts* and *{rules-ui}*, you can open the action menu (…) for the appropriate alert and select *Mute*.
```

Comment:
For Rules is "Snooze" but for Alerts is "Mute". Should we mention the different names or will it be confusing?

Reply:
The link on line 80 includes the term "snooze" in its title, but otherwise I have intentionally avoided using that term in this page.